woocommerce/woocommerce-gutenberg-products-block
# Product Gallery block: Add E2E tests to cover Variable Products

## What

This PR adds e2e tests that covers the functionality of replacing the Large Image with the selected variable product from the Add to Cart With Options block.

Fixes #11459

## Why


When I was working on the issue to add support for variation image updates I decided to handle the e2e tests for that functionality in a later PR, so this is the PR to handle that.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Ensure that the E2E tests pass on the CI.
2. You can also run `npm run test:e2e:side-effects` on your local machine and make the tests are passing.


## WooCommerce Visibility


Required:
